itest/litd_custom_channels_test.go

```diff
@@ -2637,3 +2638,87 @@ func testCustomChannelsOraclePricing(_ context.Context,
 		noOpCoOpCloseBalanceCheck,
 	)
 }
+
+// testCustomChannelsFee tests the whether the custom channel funding process
+// fails if the proposed fee rate is lower than the minimum relay fee.
+func testCustomChannelsFee(_ context.Context,
+	net *NetworkHarness, t *harnessTest) {
+
+	ctxb := context.Background()
+	lndArgs := slices.Clone(lndArgsTemplate)
+	litdArgs := slices.Clone(litdArgsTemplate)
+
+	zane, err := net.NewNode(
+		t.t, "Zane", lndArgs, false, true, litdArgs...,
+	)
+	require.NoError(t.t, err)
+
+	litdArgs = append(litdArgs, fmt.Sprintf(
+		"--taproot-assets.proofcourieraddr=%s://%s",
+		proof.UniverseRpcCourierType, zane.Cfg.LitAddr(),
+	))
+
+	charlie, err := net.NewNode(
+		t.t, "Charlie", lndArgs, false, true, litdArgs...,
+	)
+	require.NoError(t.t, err)
+	dave, err := net.NewNode(t.t, "Dave", lndArgs, false, true, litdArgs...)
+	require.NoError(t.t, err)
+
+	nodes := []*HarnessNode{charlie, dave}
+	connectAllNodes(t.t, net, nodes)
+	fundAllNodes(t.t, net, nodes)
+
+	charlieTap := newTapClient(t.t, charlie)
+	daveTap := newTapClient(t.t, dave)
+
+	// Mint an assets on Charlie and sync Dave to Charlie as the universe.
+	mintedAssets := itest.MintAssetsConfirmBatch(
+		t.t, t.lndHarness.Miner.Client, charlieTap,
+		[]*mintrpc.MintAssetRequest{
+			{
+				Asset: itestAsset,
+			},
+		},
+	)
+	cents := mintedAssets[0]
+	assetID := cents.AssetGenesis.AssetId
+
+	t.Logf("Minted %d lightning cents, syncing universes...", cents.Amount)
+	syncUniverses(t.t, charlieTap, dave)
+	t.Logf("Universes synced between all nodes, distributing assets...")
+
+	// Fund a channel with a fee rate of zero.
+	zeroLowFeeRate := uint32(0)
+
+	_, err = charlieTap.FundChannel(
```

Comment:
AFAICT this command will also fail if no feerate is specified - is that the behavior we want here?

To be clear, I think that's the behavior we have right now. Mentioning it here because that differs from what we do for asset minting or on-chain transfer, where a value of 0 feerate causes tapd to fetch a fee estimate and use that for the TX.

Reply:
I checked and your assumption is correct. Passing no feerate vs passing a feerate of zero yields the exact same result. We don't use fee estimation when funding channels. If we want that we could make a seperate issue as a feature request.